fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -853,8 +853,8 @@ private void assertMementoLink(final Link expected, final Link actual) {
 
             final var expectedUri = expected.getUri().toString();
             final var actualUri = actual.getUri().toString();
-            assertEquals(expectedUri.substring(0, expectedUri.length() - 2),
-                    actualUri.substring(0, actualUri.length() - 2));
+            assertEquals(expectedUri.substring(0, expectedUri.length() - 3),
```

Comment:
If i'm understanding correctly, this change basically increases wiggle room allowed between the expected timestamp and the real timestamp up to the minutes range? Sort of wondering if it can tell apart the 4 mementos created in that test since they are separated by 1 second each, but it seems like that already would have been an issue since it seems like it was trimming off the seconds from the timestamp.

I'm actually kind of surprised this test is passing since the logs for it are cranking out a lot of grizzly `Method Not Allowed` errors. But I can't really remember where versioning support stands in fedora 6 at the moment anyway.

Reply:
Yes, it increased the wiggle room. It's not as good as it could be if the waits were longer, but I personally don't think it's worth inserting longer waits into the tests.

The comparison behavior works like this:

1. Is the timestamp exactly what we expected?
   1. If yes, we're done -- PASS
   2. If no, continue
2. Are the `Instants` within 5 seconds of each other?
   1. If yes, continue
   2. If no, we're done -- FAIL
3. Are the first parts of the memento strings the same (this is the part I just modified)?
  1. If yes, we're done -- PASS
   2. If no, we're done -- FAIL

So, in the ideal case, the strings will match exactly. But, if they don't match, it checks to see that they're at least close and that they're formatted correctly.

Reply:
Thanks for the explanation, I hadn't followed it all the way through. That seems moderately safe, although there's obviously still some chance of false positive, but that seems outside the scope of this PR.